sim/shaman/enhancement/enhancement.go

```diff
@@ -97,6 +97,9 @@ type EnhancementShaman struct {
 	rotation Rotation
 
 	scheduler common.GCDScheduler
+
+	LavaburstWeave bool // flag to enable lava burst weaving for enh
+	WeaveLatency   int32
```

Comment:
is there a reason to put this here instead of in the other struct? ele shaman has it's spec-specific options there

Reply:
It's better for scope, only putting var at the scope they are needed and not higher up. I feel like the Shaman.go has a lot of scope issues it's where 2 Specs share things that are global among shamans but everything seems to be thrown there. used druids as a reference of keeping spec specific options in there own .go files.

Reply:
this is obsolete now that rotation options are sent directly to the rotation instead of the unit.